examples/jsm/postprocessing/SSRPass.js

```diff
@@ -382,10 +382,9 @@ SSRPass.prototype = Object.assign( Object.create( Pass.prototype ), {
 		renderer.setRenderTarget( this.beautyRenderTarget );
 		renderer.clear();
 		if ( this.groundReflector ) {
-
+			this.groundReflector.visible = false;
 			this.groundReflector.doRender( this.renderer, this.scene, this.camera );
 			this.groundReflector.visible = true;
```

Comment:
Could you please restore the empty line style too?

Reply:
no problem

Reply:
Thanks!